### ISSUE TITLE
remove seeing yourself from /look

### ISSUE THREAD

**Issue by modgethanc** [vilmibm/tildemush#99]
server should check for the sender appearing in a room contents and omit returning that in the `/look` output, since it doesn't really make sense to see yourself described in the third person in a room.

**Comment by vilmibm** [comment]
yeah, this was intentional but after seeing it in practice it's kind of silly...
